ddl/index.go

```diff
@@ -687,6 +689,14 @@ func (w *addIndexWorker) run(d *ddlCtx) {
 		}
 
 		log.Debug("[ddl-reorg] got backfill index task:#v", task)
+
+		if task.mockTask {
```

Comment:
Why not gofail here directly?

Reply:
Here is in a multi-thread environment, so it's non-trivial to control the gofail run only once.

Reply:
@winkyao 
I think it worth it to gofail here directly.
We can check the worker id to avoid race.